tools/tidy/tests/OnlyAssignedOnResetTest.cpp

```diff
@@ -0,0 +1,219 @@
+#include "../synthesis/OnlyAssignedOnReset.cpp"
```

Comment:
Should probably have an easy way to write tests against a particular check without needing to cross include the source file.

Reply:
Not an elegant solution, but how can I do that without having a header for each check?

Reply:
Perhaps you can enable the check using the registry and its string name? You don't actually need to see the class for the check, you just need to turn it on.